process/Issue_Triaging.md

```diff
@@ -0,0 +1,113 @@
+
+
+## Terminologies
+
+* TSC: Technical Steering Committee. An informal group of people who meet on a monthly basis and discuss technical topics related to software development. Everyone is welcome to join the Technical Steering Committee. Refer to the charter for more details.
+
+
+## Process
+
+The basic process of triaging issues is
+
+|First Step     |Next Step            |Result     |
+|---            |---                  |---        |
+|TSC review ->  |Accepted, or         |
+|               |Community review ->  |Closed, or |
+|               |                     |Accepted   |
+
+Please note:
+1. TSC reviewers can accept an issue but cannot close an issue before it is being reviewed by the community (the "Community Review" step).
+2. We are much less likely to accept PRs that have no associated accepted issues
+
+Issues can be at four major states during this process. They are easily identifiable:
+
+|State|What your GitHub issue looks like|
+|---|---|
+|TSC Review|matches the query `label:"tsc-review"`|
+|Community Review|matches the query `milestone:"Backlog Candidates"`|
+|Accepted|has any milestone except `Backlog Candidates`|
+|Closed|matches the query `is:closed`|
+
+In the rest of this document, we'll go into more detail about each of the activities of triaging.
+
+
+## TSC Review
+
+TSC rotates a primary and a secondary reviewer every week. The reviewers triage issues by assigning the following labels:
+
+### Categorization Labels
+
+Each issue must have a **type** label. Most type labels are grey, some are yellow. Bugs are grey with a touch of red.
+
+|Type|Description|
+|---|---|
+|[`needs-more-info`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | not possible to assign a type label due to missing information|
+|[`bug`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | the implementation of a feature is not correct|
+|[`feature-request`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | request for a new feature|
+|[`tech-debt`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | improve the implementation/architecture|
+|[`engineering`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | issues related to our engineering system or our processes|
+|[`polish`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | a feature could be improved, but not necessarily a bug|
+|[`*question`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | we should direct questions to StackOverflow|
+
+### Feature Areas Labels
+
+Each issue must have a **feature area** label. 
```

Comment:
This can get tricky. In native we also have SDK features so it may be hard to cover all areas with labels.

Reply:
Yea, I don't find this appealing as well...